Documentation/contributing/development/dev_setup.rst

```diff
@@ -381,11 +381,24 @@ httpd servers) and tests various simple network policies with
 them. These containers should be automatically removed when the test
 finishes.
 
+.. _making_changes:
+
 Making Changes
 ~~~~~~~~~~~~~~
 
-#. Create a topic branch: ``git checkout -b myBranch master``
-#. Make the changes you want
+#. Make sure the ``master`` branch of your fork is up-to-date:
+
+   ::
+
+      git fetch upstream master:master
+
+#. Create a PR branch with a descriptive name, branching from ``master``:
+
+   ::
+
+      git switch -c pr/changes-to-something master
+
```

Comment:
how is this different from `git checkout -b myBranch master`?

Reply:
It has no difference in effect, the only difference is that `git switch` is preferred over `git checkout` since Git 2.23: https://github.com/git/git/blob/master/Documentation/RelNotes/2.23.0.txt#L61-L65

Reply:
I think this is no different. The `git checkout` command is ambiguous in that it serves multiple purposes, in particular changing branch _or_ restoring files. So some time ago git received two new commands, `git switch` (to change branch) and `git restore` (to restore files, no kidding), to make it more friendly to newcomers. I believe `git switch` is now the recommended way to change branch, `git switch -c` option behaves like `git checkout -b` as far as I understand.